### PR TITLE
[docs] Example of using official prometheus helm chart to deploy server with sidecar

### PR DESCRIPTION
**Pull request by homelessnessbo**
A lot of people out there use helm as the main packaging system for deploying k8s stuff and I got a couple of requests for sharing a deploying code, so I thought it'd be nice just to put an example to the documentation.

### CODE REVIEW

**Comment by homelessnessbo** [comment]
@povilasv I wish I wasn't either. Who shell we ping to merge this?

**Comment by povilasv** [comment]
@homelessnessbo we have a #thanos-prs channel fill free to post it there :)

File: tutorials/kubernetes-demo/manifests/prometheus-helm-chart-with-sidecar.yaml [hunk 1]
```diff
@@ -0,0 +1,154 @@
+To deploy thanos sidecar along with Prometheus using official helm chart - just run the next command, putting the values to a file `values.yaml` and changing `--namespace` value beforehand:
```
Comment (bwplotka) [inline tutorials/kubernetes-demo/manifests/prometheus-helm-chart-with-sidecar.yaml]: This is definitely not a yaml file. It contains yaml, sure but it's not a YAML not a manifest either.

Can we put it in some new directory like `tutorials/kubernetes-helm/README.md` ? (: I think that would make more sense.

**Comment by homelessnessbo** [comment]
@bwplotka yeah, I also never recommend using helm to anyone though I'm in a helm trap myself. And, like we or not, people use helm, they use it a lot. And it's better to give people using helm some point to start their journey then to make every helm+thanos user go through the struggle of inventing another bicycle.

File: tutorials/kubernetes-demo/manifests/prometheus-helm-chart-with-sidecar.yaml (hunk 1 above)
Comment (homelessnessbo) [reply tutorials/kubernetes-demo/manifests/prometheus-helm-chart-with-sidecar.yaml]: @bwplotka changes applied

**Pull request merged**